vm-setup/roles/v1aX_integration_test/templates/cluster-template-controlplane-kubeadm-config-centos.yaml

```diff
@@ -0,0 +1,128 @@
+users:
+- name: {{ IMAGE_USERNAME }}
+  sudo: \"ALL=(ALL) NOPASSWD:ALL\"
+  sshAuthorizedKeys:
+  - {{ SSH_PUB_KEY_CONTENT }}
+preKubeadmCommands:
+  - systemctl restart NetworkManager.service
+  - ifup eth0
+  - systemctl enable --now crio keepalived kubelet
+  - systemctl link /lib/systemd/system/monitor.keepalived.service
+  - systemctl enable monitor.keepalived.service
+  - systemctl start monitor.keepalived.service
+postKubeadmCommands:
+  - mkdir -p /home/{{ IMAGE_USERNAME }}/.kube
+  - cp /etc/kubernetes/admin.conf /home/{{ IMAGE_USERNAME }}/.kube/config
+  - chown {{ IMAGE_USERNAME }}:{{ IMAGE_USERNAME }} /home/{{ IMAGE_USERNAME }}/.kube/config
```

Comment:
I realize there was no change here, but I'm curious why this is needed. Don't we get the kubeconfig from a Secret in the management cluster?

Reply:
All these commands are happening on the target node. Once the target cluster is provisioned and kubeadm is done, you can `ssh metal3@192.168.111.100` and run `kubectl` commands to query the target Kubernetes cluster on the target node.

Reply:
Right, so it's more of a convenience than anything else?

Reply:
Yes, I don't think there's anything in CI that depends on `kubectl` commands run on the target node as the metal3 user. It's for when you SSH into the node and want to see what the pods are doing.